packages/venia-concept/src/components/SearchBar/suggestedProduct.js

```diff
@@ -10,15 +10,17 @@ const productUrlSuffix = '.html';
 
 class suggestedProduct extends Component {
     static propTypes = {
-        handleOnProductOpen: PropTypes.func.isRequired,
-        url_key: PropTypes.string.isRequired,
-        small_image: PropTypes.string.isRequired,
-        name: PropTypes.string.isRequired,
-        price: PropTypes.object.isRequired,
-        classes: PropTypes.shape({
-            root: PropTypes.string,
-            productName: PropTypes.string,
-            productImage: PropTypes.string
+        handleOnProductOpen: func.isRequired,
+        url_key: string.isRequired,
+        small_image: shape({
+            url: string.isRequired
+        }),
```

Comment:
```suggestion
        }).isRequired,
```

Because you're accessing `small_image.url`, `render()` will throw if it's not an object.

Reply:
Added